hello-world-notification-microservice/README.md

```diff
@@ -72,4 +72,43 @@ we will receive the following message as we have specified to transform the filt
   "source": {"id":"<DEVICE ID>"},
   "type": "c8y_Speed"
 }
-```
\ No newline at end of file
+```
+
+## Device creation and measurement sending
+
+The included [Python script](demo/measurements.py) has everything to get you started with the microservice and has example code for device creation, deletion and sending out measurements.
```

Comment:
I'd use a full-stop and new sentence rather than a comma; That is;' To use the script you will need to provide authentication details. Edit the script and provide values to the following fields;'